src/main/java/core/basesyntax/report/ReportMakerImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.report;
+
+import core.basesyntax.csv.writer.WriterService;
+import core.basesyntax.csv.writer.WriterServiceImpl;
+import core.basesyntax.model.Fruit;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+public class ReportMakerImpl implements ReportMaker {
+    WriterService writerService;
+
+    public ReportMakerImpl() {
+        writerService = new WriterServiceImpl();
+    }
```

Comment:
it is not good that another service is dependent on `writer`
we should not bind the services - we should call them one by one in `main()` method